R/net.utils.R

```diff
@@ -197,23 +197,75 @@ color_tea <- function(nd, old.var = "testatus", old.sus = "s", old.inf = "i",
 #' @export
 #'
 copy_toall_attr <- function(dat, at, nwterms) {
-  otha <- names(dat$nw$val[[1]])
-  otha <- otha[which(otha %in% nwterms)]
-  if (length(otha) > 0) {
-    for (i in seq_along(otha)) {
-      va <- get.vertex.attribute(dat$nw, otha[i])
-      dat$attr[[otha[i]]] <- va
-      if (at == 1) {
+
```

Comment:
what role does the nwterms arg play, given that we want to copy all attr from nw to attr?

Reply:
In the CRAN version of ```get_attr_prop```, there is an input parameter ```only.formula``` to limit updating the attribute list to this attributes in the formation model (```fterms```). This would distinguish between the two. Can change this to all attributes as the only case and remove that language.

Reply:
Just to be clear, if there is an open population model, and an attribute passed on the network that is not in the ERGM formula, these attributes should be added/updated following the same rules as attributes on the ERGM formula.